packages/logs/src/domain/networkError/networkErrorCollection.spec.ts

```diff
@@ -37,10 +36,8 @@ describe('network error collection', () => {
 
   function startCollection(forwardErrorsToLogs = true) {
     mockFetchManager = mockFetch()
-    ;({ stop: stopNetworkErrorCollection } = startNetworkErrorCollection(
-      { ...CONFIGURATION, forwardErrorsToLogs },
-      lifeCycle
-    ))
+    const { stop } = startNetworkErrorCollection({ ...CONFIGURATION, forwardErrorsToLogs }, lifeCycle)
+    registerCleanupTask(stop)
```

Comment:
This code overwrites `window.fetch` with a mock version, and then adds our fetch instrumentation to it. The change here ensures that when we tear things down, we remove the instrumentation before overwriting `window.fetch` with its original value, so that cleanup happens correctly.